### PR TITLE
Checksum source tarballs during bootstrap.

### PR DESCRIPTION
**Pull request by stikonas**
(This is in addition to check that rootfs.py does)

### CODE REVIEW

**Comment by doraskayo** [comment]
This should prevent misaligned sources for bootstraps that don't rely on `rootfs.py`.

Very useful. Thanks!

File: sysa/SHA256SUMS.sources [hunk 1]
```diff
@@ -8,99 +8,54 @@ e1035aa2c21fae2a934d1ab56c774ce9d22717881dab8a1a5b16d294fb793489  autoconf-2.57.
...
 84d2079a20ba32f5e2d5bc79a5dcb1de94d0176c67d75d5a20d533ea6c90d691  linux-headers-5.10.41.tar.gz
 093c993767f563a11e41c1cf887f4e9065247129679d4c1e213d0544d16d8303  m4-1.4.7.tar.gz
 64b30b41fde2ebf669e6af489883fb1df6a06ac30555a96cfa3c39ecce7267dd  make-3.80.tar.gz
 3d991b33e604187c5881a0abc2e102d5b9776da5569640e73778f85d617242e7  make-3.82.tar.gz
-e40b8f018c1da64edd1cc9a6fce5fa63b2e707e404e20cad91fbae337c98a5b7  make-4.2.1.tar.gz
 e56c9463ae649d5863df3526e0af631894e0f01cdbb02a46d0db415518450dc9  mes-0.24.tar.gz
-17503d2c395dfcf106b622dc142683c1199431d095367c6aacba6eec30340459  mpc-1.2.1.tar.gz
-0c98a3f1732ff6ca4ea690552079da9c597872d30e96ec28414ee23c95558a7f  mpfr-4.1.0.tar.xz
 1370c9a812b2cf2a7d92802510cca0058cc37e66a7bedd70051f0a34015022a3  musl-1.1.24.tar.gz
+9b969322012d796dc23dda27a35866034fa67d8fb67e0e2c45c913c3d43219dd  musl-1.2.2.tar.gz
```
Comment (fosslinux) [inline sysa/SHA256SUMS.sources:48]: Should not be added

**Review by fosslinux — APPROVED** [review]
Apart from that one thing, no problems

File: sysa/SHA256SUMS.sources (hunk 1 above)
Comment (stikonas) [reply sysa/SHA256SUMS.sources]: fixed

**Pull request merged**